actors/common/src/main/java/org/sunbird/actor/user/course/dao/impl/UserCourseDaoImpl.java

```diff
@@ -0,0 +1,45 @@
+package org.sunbird.actor.user.course.dao.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections.CollectionUtils;
+import org.sunbird.actor.user.course.dao.UserCourseDao;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.models.user.courses.UserCourses;
+
+public class UserCourseDaoImpl implements UserCourseDao {
+
+  private CassandraOperation cassandraOperation = ServiceFactory.getInstance();
+  private ObjectMapper mapper = new ObjectMapper();
+  private static final String KEYSPACE_NAME = "sunbird";
+  private static final String TABLE_NAME = "user_courses";
+
+  @Override
+  public UserCourses read(String id) {
+    Response response = cassandraOperation.getRecordById(KEYSPACE_NAME, TABLE_NAME, id);
+    List<Map<String, Object>> list = (List<Map<String, Object>>) response.get(JsonKey.RESPONSE);
+    if (CollectionUtils.isEmpty(list)) {
+      return null;
+    }
+    try {
+      String jsonString = mapper.writeValueAsString((Map<String, Object>) list.get(0));
```

Comment:
We can use mapper.convertValue instead of mapper.writeValueAsString and mapper.readValue.

e.g. return mapper.convertValue((Map<String, Object>) list.get(0)), UserCourses.class);